### ISSUE TITLE
UX issue: length of career path

### ISSUE THREAD

**Issue by xavyjs** [jeenyjinjin/ChickenDinner#9]
![career_pathway](https://user-images.githubusercontent.com/5502309/52198071-a03ef300-289c-11e9-94d9-bee2e726e03c.png)
Did you try to handle career paths of more than 3 jobs which don't fit in a single line? If it extends how will the story of sequential ordering be conveyed to the user?

**Comment by haonan62** [comment]
yes, when the careers in the career path are more than 1, the system automatically goes to the next line thanks to back end assistance.
![image](https://user-images.githubusercontent.com/31196920/52909169-a9798800-32be-11e9-95e1-c0a5b1b3f238.png)
